http/http_test.go

```diff
@@ -34,3 +34,31 @@ func TestError(t *testing.T) {
 	require.Equal(t, nethttp.MethodGet, httpError.Method)
 	require.Equal(t, "/eth/v1/beacon/genesis", httpError.Endpoint)
 }
+
+func TestClientShouldSendExtraHeadersWhenProvided(t *testing.T) {
+	authroizationHeader := "Authorization"
```

Comment:
Please fix spelling here and elsewhere 'authroization' is present.

Reply:
Fixed.